Updates for Beta IDE (#1870)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/cloud-ide/ide-beta.md

The dbt Cloud integrated development environment (IDE) is where you can build, test, run, and version control your dbt projects directly from your browser. The IDE is the fastest and most reliable way to deploy dbt, and provides a real-time editing and execution environment for your dbt project -- no command line use required.

To develop in dbt Cloud IDE (beta), you need to meet these requirements:

- Your dbt project must be compatible with dbt v0.15.0. The dbt IDE is powered by the [dbt-rpc](reference/commands/rpc) which was overhauled in dbt v0.15.0.
- You must have a [Developer License](dbt-cloud/access-control/cloud-seats-and-users).  

> - Currently only multi-tenant instances of dbt Cloud can develop in the updated beta version of the Cloud IDE. Single tenant instance will see this Beta release soon.

Author: ```suggestion
- Currently only multi-tenant instances of dbt Cloud can develop in the updated beta version of the Cloud IDE. Single-tenant instances will soon be able to opt into this Beta release.
```
